Rollbar/ILogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Rollbar
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    public interface ILogger
+    {
+        ILogger Log(ErrorLevel level, object obj);
+        ILogger Log(ErrorLevel level, string msg);
+
+        ILogger LogCritical(string msg);
```

Comment:
These should just be `Critical`, `Error`, `Warning`, etc. without the `Log` prefix